parts/kubernetesagentvars.t

```diff
@@ -1,7 +1,7 @@
     "{{.Name}}StorageAccountOffset": "[mul(variables('maxStorageAccountsPerAgent'),variables('{{.Name}}Index'))]",
     "{{.Name}}Count": "[parameters('{{.Name}}Count')]",
     "{{.Name}}AvailabilitySet": "[concat('{{.Name}}-availabilitySet-', variables('nameSuffix'))]",
-    "{{.Name}}StorageAccountsCount": "[add(div(variables('{{.Name}}Count'), variables('maxVMsPerStorageAccount')), mod(add(mod(variables('{{.Name}}Count'), variables('maxVMsPerStorageAccount')),2), add(mod(variables('{{.Name}}Count'), variables('maxVMsPerStorageAccount')),1)))]",
+    "{{.Name}}StorageAccountsCount": "[variables('maxStorageAccountsPerAgent')]",
```

Comment:
Why are you changing this formula?  For example, why would we create 5 storage accounts if there is only 3 agents in the availability sets?  Note that due to a limitation the other templates use VMSS which require 5 storage accounts up front.   A customer has a limited quota on storage accounts.